### DISCUSSION TITLE
Jaeger Service/Operation to use with Grafana dashboard variables

### DISCUSSION THREAD

**Discussion by tbbream** [grafana/grafana#53011] [Dashboards, Panels & Visualizations]
Allow the service and operation to reference a Grafana dashboard variable. This will allow for a dashboard to be built that can key different information together. This can combine graphs with a table of jaeger spans that can tie all information together.

**Comment by mageddo** [comment]
Can't believe it doesn't exists yet

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
